R/utils-s3.R

```diff
@@ -76,21 +78,23 @@ s3_register <- function(generic, class, method = NULL) {
 
 .rlang_s3_register_compat <- function(fn, try_rlang = TRUE) {
   # Compats that behave the same independently of rlang's presence
-  out <- switch(fn,
+  out <- switch(
+    fn,
     is_installed = return(function(pkg) requireNamespace(pkg, quietly = TRUE))
   )
 
   # Only use rlang if it is fully loaded (#1482)
-  if (try_rlang &&
-    requireNamespace("rlang", quietly = TRUE) &&
-    environmentIsLocked(asNamespace("rlang"))) {
-    switch(fn,
-      is_interactive = return(rlang::is_interactive)
-    )
+  if (
```

Comment:
This file should be exempt from styling. I wonder if this is now in an "rlang standalone" file, these are exempt by default. Let's revert here and review later.